### PR TITLE
Kafka source support and minor fixes

### PR DESCRIPTION
**Pull request by skonto**
- Adds support for the kafka source in our dashboards
- Updates dashboards titles to include more useful info for the end user.
- Fixes an issue with ping source which has an adapter that resides in the knative-eventing ns and was not being detected.
- Aligns all eventing dashboards to have a namespace template var so user can focus in a given namespace.

Screenshots:
Dashboard grafana-dashboard-definition-knative-eventing-source:
image

image

image

image

Dashboard grafana-dashboard-definition-knative-eventing-resources : 

image

This was tested using this [repo](https://github.com/skonto/ocp-serverless-test-observability) to create sources.

/cc @aliok pls review.

### CODE REVIEW

File: knative-operator/deploy/resources/dashboards/grafana-dash-knative-eventing-resources.yaml [hunk 1]
```diff
@@ -64,7 +64,7 @@ data:
           "steppedLine": false,
           "targets": [
             {
-              "expr": "sum(rate(container_cpu_usage_seconds_total{namespace=\"$namespace\", pod=~\"$sprefix-$source-.+\", container != \"POD\", container != \"\"}[1m])) by (container)",
+              "expr": "sum(rate(container_cpu_usage_seconds_total{namespace=\"$namespace\", pod=~\"$sprefix-$source-.+|pingsource-mt-adapter-.+\", container != \"POD\", container != \"\"}[1m])) by (container)",
```
Comment (aliok) [inline knative-operator/deploy/resources/dashboards/grafana-dash-knative-eventing-resources.yaml:67]: @skonto can you explain the change here? regex here is hard to read

Comment (skonto) [reply knative-operator/deploy/resources/dashboards/grafana-dash-knative-eventing-resources.yaml:67]: Yes I need to monitor the pingsource-mt-adapter in knative-eventing namespace.
This cannot be covered by the pattern for the other resources which is:
```
$sprefix-$source-
apiserversource-testevents-f61cc82f-11e2-41d6-9ce9-9242a3cs29cm
sprefix = apiserversource
source = testevents
```

Comment (aliok) [reply knative-operator/deploy/resources/dashboards/grafana-dash-knative-eventing-resources.yaml:67]: thanks

**Comment by aliok** [comment]
sorry pressed the wrong button :(

**Pull request merged**